Sources/Sentry/SentryAppStartTrackingIntegration.m

```diff
@@ -0,0 +1,70 @@
+#import "SentryAppStartTrackingIntegration.h"
+#import "SentryAppStartTracker.h"
+#import "SentryDefaultCurrentDateProvider.h"
+#import "SentryLog.h"
+#import <Foundation/Foundation.h>
+#import <SentryAppStateManager.h>
+#import <SentryClient+Private.h>
+#import <SentryCrashAdapter.h>
+#import <SentryDispatchQueueWrapper.h>
+#import <SentryHub.h>
+#import <SentrySDK+Private.h>
+#import <SentrySysctl.h>
+
+@interface
+SentryAppStartTrackingIntegration ()
+
+#if SENTRY_HAS_UIKIT
+@property (nonatomic, strong) SentryAppStartTracker *tracker;
+#endif
+
+@end
+
+@implementation SentryAppStartTrackingIntegration
+
+- (void)installWithOptions:(SentryOptions *)options
+{
+#if SENTRY_HAS_UIKIT
+    if (options.enableAppStartMeasuring) {
```

Comment:
H: we should not rely only on this flag, but also if traces sample rate or traces sampler are set, otherwise no transactions will exist anyway